charts/kubewarden-controller/questions.yaml

```
@@ -30,6 +30,15 @@ questions:
     allow viewing secrets, roles or role bindings), and "audit-scanner-cluster-role"
     (allows read-write to Kubewarden resources and PolicyReports).
   group: "Features"
+- variable: "auditScanner.policyReporter"
+  type: boolean
+  default: false
+  show_if: auditScanner.enable=true
+  label: Enable Policy Reporter
+  description: |
+    Policy reporter is an UI to visualize the reports generated by Background
```

```suggestion
    Policy reporter is a UI to visualize the reports generated by Background
```

since it starts with consonant sound ("yu"). Although I doubt myself, @jhkrug could you chime in please? :)

---

I think @viccuad is right about his suggestion